<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>49</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>851</width>
    <height>49</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>851</width>
    <height>49</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>^</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V````</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> ctg</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Serif&quot;;</string>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoNone</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Serif&quot;;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
